@types/discordBot.d.ts

```diff
@@ -0,0 +1,70 @@
+import { BotErrorType, IdType } from '../helpers/discordBot'
+
+export type snowflake = string
```

Comment:
I don't understand this `snowflake` type. Why not use string instead?

Reply:
It's just for convenience, to make it clear at a glance that it'll be something like '828783458469675019' and not an arbitrary string.

Reply:
It's just an type alias. This way we can know that this string type actually has a snowflake id format.

Reply:
TIL about snowflake ids. This wiki link should be in comment IMO.

Reply:
discord.js has been added as dependency, snowflake type alias is now imported from it. A comment describing it has been added in discordBot.d.ts